lib/rules/web/admin_console/4.3/storage.xyaml

```diff
@@ -121,13 +121,14 @@ check_volume_missing_on_workload_page:
     selector: *volume_info
     missing: true
 remove_volume_from_container:
+  params:
+    kebab_item: Remove Volume
+    button_text: Remove Volume
   elements:
   - selector:
       xpath: //h2[contains(., Volumes)]/following-sibling::div//tr[td[@data-test-id='name' and contains(., '<volume_name>')] and td[contains(., '<container_name>')]]//button[@data-test-id='kebab-button']
     op: click
-  - selector:
-      xpath: //button[@data-test-action='Remove Volume']
-    op: click
```

Comment:
Reuse `click_kebab_item` rule which has checked the item is enabled before clicking on it